README.md

```diff
@@ -59,3 +59,7 @@ This [awesome](https://github.com/sindresorhus/awesome) repository is maintained
 
 <!--lint ignore double-link-->
 - https://innersourcecommons.org/stories/
+
+## Other Communities
+
+- FINOS InnerSource Special Interest Group: [website](https://innersource.finos.org), [GitHub](https://github.com/finos) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation.
```

Comment:
Maybe like this to make the title a link (like in all the above items):
[FINOS InnerSource Special Interest Group](https://innersource.finos.org) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation.  [GitHub](https://github.com/finos).

Reply:
```suggestion
- [FINOS InnerSource Special Interest Group](https://innersource.finos.org) - a community of people implementing, or interested in implementing, InnerSource within their financial services organization in order to increase collaboration and remove/deal with excessive ownership issues that can stall innovation. [GitHub](https://github.com/finos).
```